src/site_adapters/hackerNews.ts

```diff
@@ -39,8 +39,8 @@ const HNAdapter = createDomScrapingAdapter({
           mainRow: el,
           detailsRow: detailsRow,
           rank: el.querySelector("span.rank"),
-          title: el.querySelector("a.storylink"),
-          link: el.querySelector("a.storylink").getAttribute("href"),
+          title: el.querySelector("a.titlelink"),
+          link: el.querySelector("a.titlelink").getAttribute("href"),
```

Comment:
this might need to be

```suggestion
          title: el.querySelector("span.titleline"),
          link: el.querySelector("span.titleline > a").getAttribute("href"),
```

now 

bump @geoffreylitt